src/libraries/System.Buffers/README.md

```diff
@@ -0,0 +1,18 @@
+# System.Buffers
+Contains types used in creating and managing memory buffers, such as those represented by `Span<T>` and `Memory<T>`.
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system.buffers.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Buffers+label%3A%22help+wanted%22+) issues.
+
+## Source
+[../../coreclr/System.Private.CoreLib/src/System](../../coreclr/System.Private.CoreLib/src/System)
+
+## Tests
+[./tests](./tests)
```

Comment:
This is not correct. System.Buffers implementation lives under `src\libraries\System.Private.CoreLib\src\System\Buffers` and tests lives under `src\libraries\System.Buffers\tests`